src/traits/networking/centralized_web_server_network.rs

```diff
@@ -91,7 +91,7 @@ impl<
     fn parse_post_message(
         &self,
         message: Message<TYPES, PROPOSAL, VOTE>,
-    ) -> SendMsg<Message<TYPES, PROPOSAL, VOTE>> {
+    ) -> Result<SendMsg<Message<TYPES, PROPOSAL, VOTE>>, CentralizedWebServerNetworkError> {
         let view_number: TYPES::Time = message.get_view_number();
 
         let endpoint = match message.clone().kind {
```

Comment:
nit (and unrelated to this PR): 
```suggestion
        let endpoint = match &message.kind {
```